Gruntfile.js

```diff
@@ -65,33 +65,14 @@ module.exports = function(grunt) {
       }
     },
     ts: {
-      'amd-es6-debug': {
```

Comment:
https://github.com/amzn/ion-js/blob/master/DEBUGGER.md

I think this is a left over, we did not use to generate maps for our standard targets.